src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -456,6 +527,86 @@ void CC32xxWiFi::wlan_mac(uint8_t mac[6])
     sl_NetCfgGet(SL_NETCFG_MAC_ADDRESS_GET, nullptr, &len, mac);
 }
 
+/*
+ * CC32xxWiFi::test_mode_start()
+ */
+void CC32xxWiFi::test_mode_start()
+{
+    os_thread_t thread;
+    os_thread_create(&thread, "sl_Task", os_thread_get_priority_max(), 2048,
```

Comment:
since thread is not used in this method it can be dropped and pass nullptr instead into os_thread_create which will use a local var instead and discard the thread handle automatically.

Reply:
Fixed.